Bottom: 4437753a9535bdd83efae1081a20ba40a02b4aa5
Top:    09356cae523fedfc8dddb05ad30e47ffb9540993
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-03-21 11:51:10 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
